pkg/controllers/managementuser/resourcequota/resource_quota_calculate_used.go

```diff
@@ -44,9 +46,14 @@ func (c *calculateLimitController) calculateResourceQuotaUsedProject(key string,
 }
 
 func (c *calculateLimitController) calculateProjectResourceQuota(projectID string) error {
-	projectNamespace, projectName := getProjectNamespaceName(projectID)
+	projectNamespace, projectName := ref.Parse(projectID)
 	project, err := c.projectLister.Get(projectNamespace, projectName)
 	if err != nil || project.Spec.ResourceQuota == nil {
+		if errors.IsNotFound(err) {
+			// If Rancher is unaware of a project, we should ignore trying to calcalate the project resource quota
```

Comment:
there is a typo here but not sure why CI is passing on this

Reply:
updated